open README file with utf-8 encoding, to avoid error like:
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 577: ordinal not in range(128)
author: Kenneth Hoste (HPC-UGent)
--- paycheck-1.0.2/setup.py.orig	2015-05-21 16:09:24.300074046 +0200
+++ paycheck-1.0.2/setup.py	2015-05-21 16:11:43.503945542 +0200
@@ -1,6 +1,7 @@
 from distutils.cmd import Command
 from distutils.core import setup
 
+import codecs
 import sys
 if sys.version_info >= (3, 0):
     try:
@@ -24,7 +25,7 @@
     name = 'paycheck',
     version ='1.0.2',
     description ='A Python QuickCheck implementation',
-    long_description = open("README.txt").read(),
+    long_description = codecs.open("README.txt", encoding='utf-8').read(),
     author ='Mark Chadwick',
     author_email ='mark.chadwick@gmail.com',
     maintainer ='Gregory Crosswhite',
